### PR TITLE
Merge in 2.6.1 changes

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Bring 2.6.1 changes into main branch

# If you had changes on main or maintenance, please make sure all merging was done correctly.

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "version": "2.6.1-SNAPSHOT",
```
Comment (t1m0thyj) [inline lerna.json:2]: Do we plan to leave the versions as "2.6.1-SNAPSHOT" for now? I guess there's not really any reason to manually bump them now 🙂 We can wait until the 2.7.0 release PR to do that.

**Review by t1m0thyj — COMMENTED** [review]
LGTM, will approve once failing tests are addressed. I believe they may be related to #2129 and @traeok is already looking into them 🙂

File: lerna.json (hunk 1 above)
Comment (traeok) [reply lerna.json:2]: > LGTM, will approve once failing tests are addressed.

Just pushed the fix for this 👍

Comment (JillieBeanSim) [reply lerna.json:2]: @t1m0thyj as long as it doesn't change anything when merged I wouldn't worry about it atm.

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**